public/index.html

```diff
@@ -20,7 +20,7 @@ <h2>Add an item</h2>
 
     <button onclick="save()" id="buttonSave">Save</button>
     <button onclick="cancel()" id = "buttonClear">Cancel</button>
-
+    <p id ="parError" class="error">Error. Please try again</p>
```

Comment:
I would suggest a little bit more descriptive error text like: Error during loading data from the server. Please try again later.